stardog/manifest.json

```
@@ -1,6 +1,6 @@
 {
   "maintainer": "support@stardog.com",
-  "manifest_version": "1.0.0",
+  "manifest_version": "1.0.1",
```

```suggestion
  "manifest_version": "1.0.0",
```
Manifest version is not related to integration version. You can add another entry to the changelog https://github.com/DataDog/integrations-extras/blob/master/stardog/CHANGELOG.md#changelog---stardog